src/AppAny.Quartz.EntityFrameworkCore.Migrations.PostgreSQL/EntityTypeConfigurations/QuartzBlobTriggerEntityTypeConfiguration.cs

```diff
@@ -1,3 +1,4 @@
+using AppAny.Quartz.EntityFrameworkCore.Migrations.Quartz;
```

Comment:
With `AppAny.Quartz.EntityFrameworkCore.Migrations` namespace for models you don't need all this usings

Reply:
ok np